components/esp_eth/src/esp_eth_phy_ksz80xx.c

```diff
@@ -20,6 +20,7 @@
 typedef enum
 {
     KSZ80XX_MODEL_NUMBER_11 = 0x11,     // KSZ8041
+    KSZ80XX_MODEL_NUMBER_13 = 0x13,     // KSZ8041
```

Comment:
Out of my interest, I read the datasheet quickly but didn't find out why there's going to be a new model ID for KSZ8041 portfolio. 
![image](https://user-images.githubusercontent.com/8869469/212222780-2db67b19-d113-4bf1-a96e-49f92037d880.png)
Maybe that's for KSZ8041RNL**I** only? If so, we'd better update the comment here to "KSZ8041RNLI" in case someone else missed the background and thought it was a typo.

Reply:
I am kind of hesitant to approve this change without any confirmation from Microchip. We need to be sure that KSZ8041RNLI really returns different model number. @jamesarm97, since you have the chip and you made that observation, could you please contact Microchip customer support and report them the chip returns different model number than defined in datasheet?

Reply:
Yes, we have this chip on a board and it will not work with the code as is. I did some debugging and it is returning 0x13. The data sheet we have is the same you are showing so we were a little confused because we picked this chip knowing it was in the list of PHYs with a driver for the ESP IDF. All other registers read correctly (acts like the x11 and x21 as far as the OP_MODE_REG and offset go.

Reply:
See also https://github.com/torvalds/linux/commit/4bd7b5127bd02c12c1cc837a7a0b6ce295eb2505

Reply:
Glad it isn't just me. That seems to match the same ID I am reading. I did send a ticket request to microchip to see what their response is and will post when/if I get something.

Reply:
Still waiting for Microchip to get back with me. They said they are very busy and will get to it. Frustrated because I am trying to use the arduino framework on ESP32 with this chip so the ESP-IDF release v4.4 also needs to be patched to work with the arduino framework since that is the highest version it works with. That version PHY code does the Model number check a little differently and isn't a clean solution like the master version. Everything I try seems to be failing  because the arduino esp framework seems to want to pull from the github instead of using my local files. If there are any pointers for that it would be appreciated.

Reply:
Here is the correspondence with Microchip so far. They are looking into getting the chip doc updated as well:

“Thank you very much for your patience. I apologize for the delay in my response; my colleagues have informed me that we can now proceed with providing technical support in this case.

Thank you for sharing screenshots taken from the Linux driver. I also checked the driver source code before and noticed the same observation you made. For reference, here is the commit referencing the message mentioned in the case description:

https://github.com/torvalds/linux/commit/4bd7b5127bd02c12c1cc837a7a0b6ce295eb2505

Out of curiosity, I went to the ESP-IDF GitHub page and noticed this pull request which I believe you submitted. Is my assumption correct?

https://github.com/espressif/esp-idf/pull/10533

Other than that, I will reach out to my colleagues in the Business Unit about this observation. I will also ask if the datasheet could be updated to reflect this. In the meantime, I believe proceeding with the change should be okay for development since the Linux driver also has the same change.”